rmf_fleet_msgs/msg/Location.msg

```diff
@@ -2,5 +2,6 @@ builtin_interfaces/Time t
 float32 x
 float32 y
 float32 yaw
+float32 approach_speed # Target speed of the lane leading to this waypoint
```

Comment:
Maybe the comment can specify what range of values will enforce speed limits? Eg: greater than zero

Reply:
@luca-della-vedova I missed this, it would be good to add a comment about limits and the units as well

Reply:
The fact that it was a limit was stated after the feedback but I added the unit (m/s) https://github.com/open-rmf/rmf_internal_msgs/pull/28/commits/b50f98319619780e36af4dd4aa6c5e841b08cf19